docs/DeveloperGuide.md

```diff
@@ -897,17 +927,17 @@ Priorities: 1 (must have), 2 (nice to have), 3 (unlikely to have)
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  Should be able to respond within 1 second for each operation.
-4.  Should be able to function fully without connecting to internet.
+3.  Should be able to respond within 2 second for each operation.
+4.  Should be able to function fully without access to internet.
 5.  Should be for a single user.
-6.  Should not handle the printing of the report (i.e. sales report).
-7.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should
+6.  Data files should remain unchanged when transferring from a computer to another.
+7.  Should not attempt to make any change in all data files.
+8.  Should not handle the printing of the report (i.e. sales report).
```

Comment:
I don't think we need to say this? It sounds like a feature to me. This can be discussed further.